dotnet-mfa-example/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,85 @@
+                <a href="https://workos.com/docs/reference" target="_blank"><button class='button'>API
+                        Reference</button></a>
+                <a href="https://workos.com/blog" target="_blank"><button class='button'>Blog</button></a>
+
+            </div>
+        </div>
+
+        <div class="logged_in_div_right">
+            <div class="flex_column">
+
+                <div class="flex">
+                    <div class="factor_card flex_column space-evenly">
+                        @using (Html.BeginForm("EnrollSmsFactor","Home", FormMethod.Post)){
+                            <div>
+                                <div class="flex">
+                                    <input class="text_input" type="text" id="phone_number" name="phone_number"
+                                        placeholder="Phone Number" required>
+                                </div>
+                                <div class="flex">
+                                    <button type="submit" name="type" value="sms"
+                                        class="button button-outline button-sm">Enroll New SMS Factor
+                                    </button>
+                                </div>
+                            </div>
+                        }
+                    </div>
+
+                    <div class="factor_card flex_column space-evenly">
+                        @using (Html.BeginForm("EnrollTotpFactor","Home", FormMethod.Post)){
+                            <div class="flex-column">
+                                <div class="flex">
+                                    <input class="text_input" type="text" id="totp_issuer" name="totp_issuer"
+                                        placeholder="TOTP Issuer" required>
+                                </div>
+                                <div class="flex">
+                                    <input class="text_input" type="text" id="totp_user" name="totp_user"
+                                        placeholder="User Email" required>
+                                </div>
+                            </div>
+                            <div class="flex">
+                                <button type="submit" name="type" value="totp"
+                                    class="button button-outline button-sm">Enroll
+                                    New TOTP Factor</button>
+                            </div>
+                        }
+                    </div>
+
+                </div>
+
+
+                <h2>MFA Factors List</h2>
+                @if (ViewBag.factors != null) {
+                    @foreach (var factor in ViewBag.factors)
+                    {
+                        <a href='@string.Format("/factor_detail/{0}", factor.Id)' class="button login_button">
+                            @factor.Type | @factor.Id
+                        </a>
+                    }
+                }
+            </div>
+        </div>
+    </div>
+</body>
```

Comment:
Github calling out that there should be a newline at the end of the file. I ran into this the other day where VS Code was automatically chopping off the new lines, so idk if it's really necessary.